src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVFileUtils.java

```diff
@@ -19,36 +20,16 @@ public static void writeSAMFile(final String outputName, final Iterator<SAMRecor
         Utils.nonNull(header, "provided header for outputting sam file is null");
         Utils.nonNull(outputName, "provided output name is null");
 
-        try (final SAMFileWriter writer = getSAMFileWriter(outputName, header, preOrdered)) {
+        final SAMFileWriterFactory factory = new SAMFileWriterFactory()
+                .setCreateIndex(preOrdered && outputName.endsWith(".bam")
```

Comment:
I'd make constants for these extensions. For bam you could use htsjdk's `BamFileIoUtils.BAM_FILE_EXTENSION`.

Reply:
done